### DISCUSSION TITLE
ump sdk has been updated with new helper methods

### DISCUSSION THREAD

**Discussion by cumul0nimbus** [admob-plus/admob-plus#603] [Q&A]
Hey @ratson first of all thanks for your awesome plugins. Are there any plans to update your consent plugin to implement new helper methods.

**Comment by 1n3JgKl9pQ6cUMrW** [comment]
There is a new (bèta) version of this plugin in the making;

https://github.com/admob-plus/admob-plus/releases/tag/cordova-plugin-consent%403.0.0-alpha.4

Maybe that's also offering new "helpers"?

_Not sure which "helpers" you mean,_

**Reply by cumul0nimbus** [reply]
I mean new methods that are added in updated ump sdk. 
https://developers.google.com/interactive-media-ads/ump/android/release-notes

**Reply by 1n3JgKl9pQ6cUMrW** [reply]
Does this alternative plugin support the helpers?

https://github.com/EMI-INDO/emi-indo-cordova-plugin-consent

**Reply by cumul0nimbus** [reply]
Unfortunately, this plugin does not accommodate the recently introduced methods in the latest release (2.1.0) from July 24, 2023. It is anticipated that this updated SDK has undergone enhancements and has addressed ad serving related issues.Please review the release notes for this SDK. The deadline for integrating this SDK is January 16, 2024. It is imperative that we promptly incorporate the consent for this SDK.

https://developers.google.com/interactive-media-ads/ump/android/release-notes

**Comment by EMI-INDO** [comment]
> Does this alternative plugin support the helpers?
> 
> https://github.com/EMI-INDO/emi-indo-cordova-plugin-consent

next release add

 https://developers.google.com/interactive-media-ads/ump/android/release-notes
